migrations/2022-12-23-143535_add_unsubscribe_token/up.sql

```diff
@@ -0,0 +1,4 @@
+-- Your SQL goes here
+ALTER TABLE users ADD COLUMN unsubscribe_token VARCHAR(32) NOT NULL UNIQUE DEFAULT substring(md5(random()::text), 0, 32);
```

Comment:
```suggestion
CREATE EXTENSION IF NOT EXISTS pgcrypto;
ALTER TABLE users ADD COLUMN unsubscribe_token VARCHAR(32) NOT NULL UNIQUE DEFAULT translate(encode(gen_random_bytes(24), 'base64'), '+/=', '-_');
```
With this method we get 24 bytes of actual secure randomness (although an unsubscribe token does not have very strict security requirements), but requires the `pgcrypto` extension to be enabled. However this only needs to be done once for the database, hence I added it to the migration.